docs/developing.md

```diff
@@ -209,3 +209,11 @@ A shortcut task available from the root also runs the above commands:
 ```bash
 yarn reset
 ```
+
+#### Yarn/Gulp tasks are failing after rebasing or getting new `master`
+
+A possiblity is that new package is added to this monorepo but `packages/newpackage` is not linked in `node_modules`. To sreolve that, you can run:
```

Comment:
```suggestion
It is possible that a newly introduced package is added to this monorepo, but `packages/newpackage` is not linked in `node_modules`. While a `yarn reset` may resolve this, another possible way to resolve this is to run::
```

@asudoh thank you for adding this! I made some word changes, and can you also run `yarn doctoc` to update the TOC on this doc?

Reply:
@jeffchew Sure, done!